core/src/templates/components/quote/quote.twig

```diff
@@ -1,8 +1,8 @@
-<div class="su-quote">
+<article class="su-quote">
```

Comment:
This passes the smell test.

Reply:
I'm ok with this one also!

Reply:
I disagree. Quotes often depend on their context for proper comprehension. I don't believe a quote would "make sense on its own in a feed reader".

Reply:
Why not `blockquote`?

Reply:
@kgcreative , I think we didn't use blockquote around the whole component itself because it has quite a bit of extra stuff - image, company/title and the name is used as a header (which Level Access asked us to change it to an actual h header). However, this makes me think that perhaps we should use blockquote for the actual quote portion of the component :)